src/Security/SecureTransport.cs

```diff
@@ -82,7 +82,21 @@ public enum SslStatus {
 		BadConfiguration			= -9848,
 		UnexpectedRecord			= -9849,
 		SSLWeakPeerEphemeralDHKey               = -9850,
-		SSLClientHelloReceived                  = -9851 // non falta
+		SSLClientHelloReceived                  = -9851, // non falta
+		SSLTransportReset                       = -9852,
+		SSLNetworkTimeout                       = -9853,
+		SSLConfigurationFailed                  = -9854,
+		SSLUnsupportedExtension                 = -9855,
+		SSLUnexpectedMessage                    = -9856,
+		SSLDecompressFail                       = -9857,
+		SSLHandshakeFail                        = -9858,
+		SSLDecodeError                          = -9859,
+		SSLInappropriateFallback                = -9860,
+		SSLMissingExtension                     = -9861,
+		SSLBadCertificateStatusResponse         = -9862,
+		SSLCertificateRequired                  = -9863,
+		SSLUnknownPSKIdentity                   = -9864,
```

Comment:
`PSK` -> `Psk` (and watch out for typo errors)

Reply:
Yeah I thought on camelcasing it but we would have `SSL` and `Psk`, `SSLUnknownPskIdentity` looks a bit weird to me. But if you really want it I'll change it heh

Reply:
too easy to miss if if we decide to `Ssl` them in `XAMCORE_4_0`